### DISCUSSION TITLE
How to hide neo-tree from tabline and statusline?

### DISCUSSION THREAD

**Discussion by careb0t** [nvim-lualine/lualine.nvim#1330] [Configuration]
I have been trying to prevent neo-tree from creating a statusline and entry on the tabline when it is toggled open, but it seems like no matter what I put into the configuration, nothing actually hides it. I have tried "neo-tree", "neotree", "Neotree", "Neo-tree", "Neotree filesystem" and none of these work. No idea what else to try.

```
disabled_filetypes = {
    statusline = { "neo-tree" },
    tabline = { "neo-tree" },
},
```
Above is what my configuration looks like (other types that actually get removed not shown)

**Comment by acidburnmonkey** [comment]
did you find the solution to this?

**Reply by lifeashansen** [reply]
> did you find the solution to this?

did you find the solution to this?

**Reply by acidburnmonkey** [reply]
> did you find the solution to this?

yeah i swapped tabline for barbar https://github.com/acidburnmonkey/dotfiles/blob/6f01658b8defc1a9a73fb382a3aeeb20621a7b9f/nvim/init.lua#L256C1-L279C1

**Comment by JosemaGIT-0702** [comment]
Good afternoon my guy, I know that it has been more than a year but I may have a solution to your problem. Where were you trying to introduce the "disabled_filetypes" option? To do it properly you had to introduce it like this (in lazyvim I mean) 

`
opts = { 
    options = { 
        disabled_filetypes = { 'neo-tree' }
    }
}
`

This may seam obvious, but I was ignoring the 'options' part and just trying to introduce it inside of opts directly.

**Reply by another32m2** [reply]
Thanks!
